requirements/default.txt

```diff
@@ -155,11 +155,9 @@ django-npm==1.0.0 \
 markus==2.1.0 \
     --hash=sha256:3408818d4d78ca0f4085c6ea32b02f4b3d2a2520b169ddec3167b50f0b7c700d \
     --hash=sha256:bfa276c69b25006e37449eed447bc27105bf6f26c9ca27d01ddefbd1077b44ea
-awscli==1.16.276 \
-    --hash=sha256:6ae2d5bcce2e16906ad34bee99f3ba3d7a1633c8f0f53ef97e7575499b5a88eb \
-    --hash=sha256:6c63da10a322f1245f90df99b549074d97b08f0e6f69d8ce7f06330b4822b792
-moto==1.3.13 \
-    --hash=sha256:95d48d8ebaad47fb5bb4233854cf1cf8523ec5307d50eb1e4017ce10f1960b66
```

Comment:
I ditched moto, removed boto, and moved boto3 to the `requirements/default.txt` file.